client/src/components/TopBar.tsx

```diff
@@ -1,18 +1,19 @@
 import { Toolbar, IconButton, AppBar, Typography } from "@mui/material";
 import AccountBoxIcon from "@mui/icons-material/AccountBox";
 import React, { FC } from "react";
+import "../styles.css";
```

Comment:
This probably works, but this is not the way to do styles with Material UI. If you make a task and link it here, we can discuss it Friday